lib/stashutils/fsi/base.py

```diff
@@ -1,12 +1,24 @@
 # -*- coding: utf-8 -*-
 """helper functions and base classes."""
 
+import sys
+
 from stashutils.fsi.errors import OperationFailure
 import random
 import os
 import time
 import stat
-import pwd
+
+
+def _get_user_info():
+    uid = os.getuid()
+    if sys.executable.find("Pythonista") >= 0:
+        gid = os.getgid()
+    else:
+        import pwd
+
+        gid = pwd.getpwuid(uid).pw_gid
```

Comment:
```suggestion
    try:
        import pwd

        gid = pwd.getpwuid(uid).pw_gid
    except ImportError:  # Pythonista
        gid = os.getgid()
```
This way if Pythonisa adds the `pwd` module, then our code uses it.

Reply:
resolved with `os`